shared/login/register/select-other-device/index.render.desktop.js

```diff
@@ -5,12 +5,24 @@ import {globalStyles, globalColorsDZ2} from '../../../styles/style-guide'
 import Container from '../../forms/container.desktop'
 import type {Props} from './index.render'
 
-const Row = ({id, name, type, onSelect}) => {
-  const iconType = type === 'mobile' ? 'fa-mobile' : 'fa-laptop'
+const Row = ({deviceID, name, type, onSelect}) => {
+  const iconType = {
+    'mobile': 'fa-mobile',
+    'computer': 'fa-laptop',
+    'paper key': 'paper-key'
+  }[type]
+
+  const onClick = e => {
+    onSelect(deviceID)
+    e && e.preventDefault()
```

Comment:
true, we could have the onclick in text be an empty function, or pass in the clickable style manually.